UnleashedRecomp/ui/options_menu.cpp

```diff
@@ -1049,31 +1050,33 @@ static void DrawConfigOption(int32_t rowIndex, float yOffset, ConfigDef<T>* conf
         }
         else if constexpr (std::is_same_v<T, float> || std::is_same_v<T, int32_t>)
         {
-            float deltaTime = ImGui::GetIO().DeltaTime;
+            float deltaTime = std::fmin(ImGui::GetIO().DeltaTime, 1f/15f);
```

Comment:
`1f/15f` is invalid syntax. Change it to `1.0f / 15.0f`.

```suggestion
            float deltaTime = std::fmin(ImGui::GetIO().DeltaTime, 1.0f / 15.0f);
```

Reply:
@hyperbx My bad, haven't used cpp in a while. Sent